_includes/templates/iot-gateway/mqtt-converter-json-config.md

```diff
@@ -6,14 +6,22 @@ Json converter is default converter, it looks for deviceName, deviceType, attrib
+| attributes:                 |                           | This subsection contains parameters of the incoming message, that will be interpreted as attributes for the device.                       |
 | ... type                    | **string**                | Type of incoming data for a current attribute.                                                                                            |
-| ... key                     | **model**                 | Attribute name, that will sends to ThingsBoard instance.                                                                                  |
+| ... key                     | **model**                 | Attribute name, that will send to ThingsBoard instance.                                                                                   |
 | ... value                   | **${sensorModel}**        | Simple JSON expression, uses for looking value in the incoming message, that will send to ThingsBoard instance as value of key parameter. |
-| timeseries                  |                           | This subsection contains parameters of the incoming message, that will be interpreted as telemetry for the device.                        |
+|                             |                           |                                                                                                                                           |
+| ... type                    | **string**                | Type of incoming data for a current attribute.                                                                                            |
+| ... key                     | **${sensorModel}**        | Simple JSON expression, uses for looking value in the incoming message, that will be used as attribute name.                              |
+| ... value                   | **on**                    | Attribute value, that will send to ThingsBoard instance.                                                                                  |
+| timeseries:                 |                           | This subsection contains parameters of the incoming message, that will be interpreted as telemetry for the device.                        |
 | ... type                    | **double**                | Type of incoming data for a current telemetry.                                                                                            |
-| ... key                     | **temperature**           | Telemetry name, that will sends to ThingsBoard instance.                                                                                  |
+| ... key                     | **temperature**           | Telemetry name, that will send to ThingsBoard instance.                                                                                   |
```

Comment:
Use "to be sent to" instead of "that will send to" here and in similar places. For example:
"Telemetry name, to be sent to ThingsBoard instance."